.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# How to contribute to the project
+
+**Did you find a bug?**
+
+* **Ensure the bug was not already reported by searching on GitHub under** [issues](https://github.com/mauricioabreu/mosaic-video/issues)
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/mauricioabreu/mosaic-video/issues/new).
+
+**Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+**Do you intend to add a new feature or change an existing one?**
+
+* Suggest your change by creating a new issue as a **feature request**.
+* Do not open an issue on GitHub until you have collected positive feedback about the change. Waiting for feedback will save you time.
```

Comment:
In `open an issue` shouldn't `open a PR` instead of issue?

Reply:
You are right, changing